src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/BlockListener.java

```diff
@@ -75,7 +75,7 @@ public void onBlockPlace(BlockPlaceEvent e) {
         }
     }
 
-    @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
+    @EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
```

Comment:
```suggestion
    @EventHandler(priority = EventPriority.HIGH, ignoreCancelled = true)
```

Reply:
Can I ask why were even changing this?

Reply:
The bug was about this running AFTER HIGHEST, which should not be done anyway. By moving it TO HIGHEST we basically didn't fix the issue but rather let randomness decide.

Reply:
In theory events should try to run on NORMAL unless there is a valid reason not to, due to this being dependent on some things to process first, there is a point to be made to move it to HIGH but HIGHEST is definitely too late.